index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  <title>My Portfolio</title>
+</head>
+<body>
+
+  <header>
+
+    <div id="settings">
+      <div id="settings-date">
+
+      </div>
+      
+      <div class="settings-fa">
+        <i class="fas fa-signal"></i>
+        <i class="fas fa-wifi"></i>
+        <i class="fas fa-battery-full"></i>
+      </div>
+    </div>
+
+    <div id="union">
+      <img src="images/Icons/Icon - Menu.png" alt="union">
+    </div>
+
+    
+  </header>
+
+  <section class="headline">
+
+    <div>
+      <img id="headline-bg" src="images/Header-llustration-desktop@2x.png" alt="background-image">
+    </div>
```

Comment:
- [ ] The `<img>` tag here is wrongly used, this is a background image and is not part of the page content, therefore what should be used to set the background is the CSS `background-image` property instead.